misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/FormatterTest.java

```diff
@@ -61,7 +61,7 @@ public void test(String source, String sourcePath) throws IOException {
         SyntaxTree syntaxTree = SyntaxTree.from(textDocument);
         SyntaxTree newSyntaxTree = Formatter.format(syntaxTree);
         Assert.assertFalse(newSyntaxTree.hasDiagnostics());
-        Assert.assertEquals(newSyntaxTree.toSourceCode(), getSourceText(assertFilePath));
+        Assert.assertEquals(newSyntaxTree.toFormatterSourceCode(), getSourceText(assertFilePath));
```

Comment:
Shall we change the actual formatter API (inside the Formatter class) output also? Since that would be what is returned to someone calling the formatter.

Reply:
Done.